src/python_testing/TC_RR_1_1.py

```diff
@@ -644,11 +644,17 @@ async def add_all_groups(self,
 
             base_groups_per_endpoint: int = math.floor(groups_per_fabric / len(group_endpoints))
             groups_remainder: int = groups_per_fabric % len(group_endpoints)
-            fabric_group_index: int = 0
+
+            group_id_list = [group_id for group_id in (group_key_map[client_idx])]
```

Comment:
```suggestion
            group_id_list = list(group_key_map[client_idx])
```
might be clearer, but either way.  And maybe call it `group_id_list_copy`?